### ISSUE TITLE
Problem loading "Manipulating Data with dplyr" lesson

### ISSUE THREAD

**Issue by GeoUTM** [swirldev/swirl#685]
Hello, I am receiving a "Error in yaml.load" when I try to load "Manipulating Data with dplyr".  Can someone advise how I can solve this?

Below is a copy+paste of the console.  Thanks in advance.

===========================================================

| Please choose a course, or type 0 to exit swirl.

1: Getting and Cleaning Data
2: Take me to the swirl course repository!

Selection: 1

| Please choose a lesson, or type 0 to return to course menu.

1: Manipulating Data with dplyr
2: Grouping and Chaining with dplyr
3: Tidying Data with tidyr
4: Dates and Times with lubridate

Selection: 1

| Attempting to load lesson dependencies...

| This lesson requires the ‘dplyr’ package. Would you like me
| to install it for you now?

1: Yes
2: No

Selection: 1

| Trying to install package ‘dplyr’ now...
also installing the dependencies ‘utf8’, ‘bindr’, ‘pillar’, ‘bindrcpp’, ‘pkgconfig’, ‘Rcpp’, ‘tibble’, ‘BH’, ‘plogr’

package ‘utf8’ successfully unpacked and MD5 sums checked
package ‘bindr’ successfully unpacked and MD5 sums checked
package ‘pillar’ successfully unpacked and MD5 sums checked
package ‘bindrcpp’ successfully unpacked and MD5 sums checked
package ‘pkgconfig’ successfully unpacked and MD5 sums checked
package ‘Rcpp’ successfully unpacked and MD5 sums checked
package ‘tibble’ successfully unpacked and MD5 sums checked
package ‘BH’ successfully unpacked and MD5 sums checked
package ‘plogr’ successfully unpacked and MD5 sums checked
package ‘dplyr’ successfully unpacked and MD5 sums checked

| Package ‘dplyr’ loaded correctly!

Error in yaml.load(readLines(con), error.label = error.label, ...) : 
  (C:/Users/parkdav4/Documents/R/win-library/3.4/swirl/Courses/Getting_and_Cleaning_Data/Manipulating_Data_with_dplyr/lesson.yaml) Invalid tag for scalar: int#base60

| Leaving swirl now. Type swirl() to resume.

**Comment by PSth77** [comment]
Hi, 
I'm getting the same error.

Tried installing swirl again. 
Tried installing lesson again using "install_from_swirl("Getting and Cleaning Data")"

===============================
| Please choose a lesson, or type 0 to return to course menu.

1: Manipulating Data with dplyr
2: Grouping and Chaining with dplyr
3: Tidying Data with tidyr
4: Dates and Times with lubridate

Selection: 1

| Attempting to load lesson dependencies...

| Package ‘dplyr’ loaded correctly!

Error in yaml.load(readLines(con), error.label = error.label, ...) : 
  (C:/Users/Admin/Documents/R/win-library/3.4/swirl/Courses/Getting_and_Cleaning_Data/Manipulating_Data_with_dplyr/lesson.yaml) Invalid tag for scalar: int#base60

| Leaving swirl now. Type swirl() to resume.

**Comment by PSth77** [comment]
Hi @GeoUTM,
Try the below from Coursera forum. 
[https://www.coursera.org/learn/data-cleaning/discussions/weeks/3/threads/Ui0zUx1gEeiyDhJtnKX-OA/replies/4w5NBiI7EeiClxJAanax8A](url)

**Comment by atalv** [comment]
I also encountered similar problem, but got a workaround, please see [https://github.com/swirldev/swirl_courses/issues/391](https://github.com/swirldev/swirl_courses/issues/391)

**Comment by itsamermo** [comment]
I tried updating lesson.yaml file as per direction on link provided by @atalv . But issue remains albeit a different error like below. Pasting from console in Rsudios.
..
.
| Please choose a course, or type 0 to exit swirl.

 1: Data Analysis
 2: ErrFind
 3: Exploratory Data Analysis
 4: Getting and Cleaning Data
 5: Mathematical Biostatistics Boot Camp
 6: Open Intro
 7: R Programming
 8: R Programming Alt
 9: Regression Models
10: Statistical Inference
11: Writing swirl Courses
12: Take me to the swirl course repository!

Selection: 4

| Please choose a lesson, or type 0 to return to course menu.

1: Manipulating Data with dplyr
2: Grouping and Chaining with dplyr
3: Tidying Data with tidyr
4: Dates and Times with lubridate

Selection: 1

| Attempting to load lesson dependencies...

| Package ‘dplyr’ loaded correctly!

Error in .Call(C_unserialize_from_yaml, string, as.named.list, handlers,  : 
  Incorrect number of arguments (6), expecting 4 for 'unserialize_from_yaml'

| Leaving swirl now. Type swirl() to resume.

>

**Comment by isabentho** [comment]
@itsamermo also getting this error.. did you manage to work it out?

**Comment by itsamermo** [comment]
yes. I had to re-install lessons.yaml file from verion 2.1.16 i think.
